androidApp/src/androidMain/kotlin/org/jetbrains/kotlinconf/android/MainActivity.kt

```diff
@@ -20,7 +25,9 @@ class MainActivity : ComponentActivity() {
         )
 
         setContent {
-            App(context)
+            Box(Modifier.windowInsetsPadding(WindowInsets.systemBars)) {
```

Comment:
Adding some basic insets here just to make buttons clickable, we'll need to do proper insets work later

Reply:
I pushed a commit to this branch to fix the insets, as they bothered me for some time already